### PR TITLE
New design homepage

### PR DESCRIPTION
**Pull request by septs**

### CODE REVIEW

**Comment by neruthes** [comment]
image

All the same image...?

**Comment by neruthes** [comment]
image

Use chevron-down instead of chevron-up.

**Comment by septs** [comment]
wait

**Comment by septs** [comment]
> image
> 
> Use chevron-down instead of chevron-up.

@OiCkilL

**Comment by neruthes** [comment]
image

This padding could be improved. Try making a 20px-width gap.

**Comment by neruthes** [comment]
image

Not clickable. And cursor should be pointer.

**Comment by septs** [comment]
> Not clickable. And cursor should be pointer.

The function unavailable, the area is HOLDER!

File: index.html [hunk 1]
```diff
@@ -1,1074 +1,303 @@
...
-                color: #FFF;
-                background: #0B1117;
-            }
-            footer[data-theme="red"] {
-                color: #FFF000;
-                background: #EE0000;
-                background: #CC0000;
-            }
-            </style>
-
```
Comment (neruthes) [inline index.html]: `same timethey make` => `same time, they make`

**Pull request merged**